README.md

```diff
@@ -1,24 +1,152 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<!-- TABLE OF CONTENTS -->
 
-Things you may want to cover:
+# 📗 Table of Contents
 
-* Ruby version
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  <!-- - [🚀 Live Demo](#live-demo) -->
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  <!-- - [Deployment](#triangular_flag_on_post-deployment) -->
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-* System dependencies
+<!-- PROJECT DESCRIPTION -->
 
-* Configuration
+# 📖 Budd-Get <a name="about-project"></a>
 
-* Database creation
+**Budd-Get** is a ruby on rails app that helps you track your expenses organized by categories.
 
-* Database initialization
+## 🛠 Built With <a name="built-with"></a>
 
-* How to run the test suite
+- `Ruby on Rails`
 
-* Services (job queues, cache servers, search engines, etc.)
+<!-- Features -->
 
-* Deployment instructions
+### Key Features <a name="key-features"></a>
 
-* ...
+- Show the list of categories added
+- Add new categories
+- Show the list of transactions by category
+- Add new transactions
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://budd-get.onrender.com/)
```

Comment:
Your video presentation is over the maximum length of 5 minutes 

- [ ] Kindly record a new one and keep it within the limit